src/lib/holocene/input/chip-input.svelte

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+  import Chip from '$holocene/chip.svelte';
+
+  export let id: string;
+  export let value: string;
+  export let label = '';
+  export let placeholder = '';
+  export let name = id;
+  export let disabled = false;
+  export let required = false;
+  export let hintText = '';
+  export let validator: (value: string) => boolean = () => true;
+
+  let chips: string[] = [];
+  let displayValue: string = '';
+
+  $: invalid = false;
+  $: {
+    invalid = chips.some((chip) => !validator(chip));
+    value = chips.join(',');
```

Comment:
Wondering if callers should just bind to an array of strings instead of a comma separated string.

i.e.
`<ChipInput bind:chips={['a', 'b', 'c']} />`
instead of
`<ChipInput bind:value="a,b,c" />`